package.json

```
@@ -60,7 +61,9 @@
     "watch": "npx tsc -w",
     "lint": "npx eslint ./src --ext .ts",
     "bindings": "./scripts/genbindings.sh deps/erdstall-contracts",
-    "test": "env TS_NODE_PROJECT=\"tsconfig.test.json\" mocha --timeout 15000 -r jsdom-global/register -r ts-node/register -r tsconfig-paths/register 'src/**/*.spec.ts'"
+    "test": "env TS_NODE_PROJECT=\"tsconfig.test.json\" mocha --timeout 15000 -r jsdom-global/register -r ts-node/register -r tsconfig-paths/register 'src/**/*.spec.ts'",
+    "devpub": "npm run build && npm run postpublish",
+    "postpublish": "npx ts-node ./scripts/publish_posthook.ts './package.json' './dist/package.json'"
```

you should be able to omit the `'`s
```suggestion
    "postpublish": "npx ts-node ./scripts/publish_posthook.ts ./package.json ./dist/package.json"
```